### ISSUE TITLE
[Bug]: Search queries get stuck

### ISSUE THREAD

**Issue by ozgeklue** [milvus-io/milvus#17139]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: 2.0.2
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

As requested by Jack, I'm creating this issue to be able discuss further.

- `Search queries get stuck:`
  - Some of these problems are due to subscriptions in Pulsar getting full. We get `Backlog quota exceeded` error and we are able to fix it by clearing the subscriptions of `persistent://public/default/by-dev-search-0` topic. However, we don't know where the bottleneck is. It would be great if you have suggestions to look for for a permanent fix.
  - It's also possible that search queries get stuck without `Backlog quota exceeded` error. Indeed we are not sure how to approach this issue as we could not find a related log in Milvus pods.

### Expected Behavior

To be able to find related logs and a fix.

**Comment by yanliang567** [comment]
@ozgeklue thank you for the issue. could you please help to collect the completed milvus logs for investigation. You can use this [script](https://github.com/milvus-io/milvus/tree/master/deployments/export-log) to export Milvus logs

/assign @ozgeklue 
/unassign

**Comment by xiaofan-luan** [comment]
> 

Gotca, this issue should be fixed by 2.1 since we removed the search channel.
For now,  you can simply clean the search channel backlog. It's due we forget to unregister  pulsar consumer  of search channel.
To make the fix clean, you should also check if any consumer of this channel is not alive(nodeID should help)
